external/src/test/groovy/org/hawkular/alerts/rest/LifecycleITest.groovy

```diff
@@ -325,8 +326,9 @@ class LifecycleITest extends AbstractQuarkusITestBase {
     }
 
     @Test
+    @Disabled
```

Comment:
should we delete the test instead of disabling it?

Reply:
I'd rather keep it here for now. I disabled the triggers history Infinispan persistence in Hawkular using a property. Nothing got deleted or changed for now. This test would still be relevant if we decided to change the property value for some reason.

Reply:
I don't like to keep unused lines of code because I think this code maybe never won't be used in the future and people will still have to read it and wonder why this is disabled.

If you would like to keep it, putting a reason @Disabled("disabled because of xyz") would be great so everyone knows why it's disabled.

Reply:
Good point, thanks. I added the reason.

Reply:
Great. Thanks for adding it :)